hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1365,6 +1365,13 @@ message DeleteKeyRequest {
 
 message DeleteKeysRequest {
     optional DeleteKeyArgs deleteKeys = 1;
+    optional RequestSource sourceType = 2 [default = USER];
+}
+
+enum RequestSource {
+  USER = 1;
+  RETENTION = 2;
```

Comment:
Can wen change the RETENTION to LIFECYCLE?

Reply:
updated.